enterprise/management/commands/email_drip_for_missing_dsc_records.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+"""
+Django management command for sending an email to learners with missing DataSharingConsent records.
+"""
+import logging
+from datetime import timedelta
+
+from django.core.management import BaseCommand
+from django.utils import timezone
+
+from consent.models import DataSharingConsent, ProxyDataSharingConsent
+from enterprise import utils
+from enterprise.models import EnterpriseCourseEnrollment
+
+LOGGER = logging.getLogger(__name__)
+
+PAST_NUM_DAYS = 1
+
+
+class Command(BaseCommand):
+    """
+    Django management command for sending an email to learners with missing DataSharingConsent records
+    """
+
+    def handle(self, *args, **options):
+        current_minus_two_days = timezone.now().date() - timedelta(days=PAST_NUM_DAYS)
+
+        enterprise_course_enrollments = EnterpriseCourseEnrollment.objects.filter(
```

Comment:
It would be better to prefetch `enterprise_customer_user` here to avoid multiple queries inside the loop later.

Reply:
@irfanuddinahmad  we can filter out yesterday's records to avoid overlapping issues.
```
yesterday = datetime.date.today() - datetime.timedelta(days=1)
EnterpriseCourseEnrollment.objects.filter(created__date=yesterday)
```

Reply:
Done